Libraries/LibJS/AST.cpp

```diff
@@ -1297,6 +1296,46 @@ Value TemplateLiteral::execute(Interpreter& interpreter) const
     return js_string(interpreter, string_builder.build());
 }
 
+void TaggedTemplateLiteral::dump(int indent) const
+{
+    ASTNode::dump(indent);
+    print_indent(indent + 1);
+    printf("(Tag)\n");
+    m_tag->dump(indent + 2);
+    print_indent(indent + 1);
+    printf("(Template Literal)\n");
+    m_template_literal->dump(indent + 2);
+}
+
+Value TaggedTemplateLiteral::execute(Interpreter& interpreter) const
+{
+    auto tag = m_tag->execute(interpreter);
+    if (interpreter.exception())
+        return {};
+    // FIXME: We really should have a helper for this.
+    if (!tag.is_object() || !tag.as_object().is_function()) {
```

Comment:
How about we just add `Value::is_function()`? :)

Reply:
Yes! It was about time :)